### PR TITLE
Remove GRUB support from lib/Update.py

### PR DESCRIPTION
**Pull request by kmoore134**
Ticket: #27928

### CODE REVIEW

**Comment by kithrup** [comment]
Where's the rest of the changes to remove grub?   They need to go in before this.  (That is, it doesn't matter if the update code mounts/unmounts /boot/grub if it's not needed, but it breaks things if it dooesn't do that and it _is_ needed.)

**Comment by kmoore134** [comment]
Understood - Please check rocket chat, since I'm looking for the place to best do that now ;)

File: lib/Update.py [hunk 1]
```diff
@@ -641,36 +634,19 @@ def MountClone(name, mountpoint=None):
         return None
 
     # If all that worked... we now need
-    # to set up /dev, /var/tmp, and /boot/grub.
+    # to set up /dev, /var/tmp
     # Let's see if we need to do that
```
Comment (amotin) [inline lib/Update.py:638]: This comment line should also be removed probably.

**Review by amotin — APPROVED** [review]
First time see that code, but nothing significant got into my eye except one extra comment line and few things that look broken, but luckily going to be removed.

**Pull request merged**